### ISSUE TITLE
403s when manually refreshing screens

### ISSUE THREAD

**Issue by Terrance** [slapperwan/gh4a#1314]
I've handed OctoDroid a [fine-grained personal access token](https://github.com/settings/tokens?type=beta) covering all the features I can see in the README, and most screens (except Notifications as apparently they're not supposed by the new tokens yet) seem to work when selected from the sidebar, but doing a pull-to-refresh on them causes a 403 screen to appear:

> Resource not accessible by personal access token (403)

This 403 also shows up in the news feed when first opening the app, but changing screens from the side menu and changing back again causes it to load fine.  Is there an additional request being made on manual refreshes?  I can't see URLs being logged anywhere (stack traces in logcat just log `Loading data failed` and the reason alongside the stack), so I can't tell what's actually being requested.

**Comment by maniac103** [comment]
No, pull-to-refresh just restarts the loaders, that is, triggers the same kind of data fetch as done when initially opening the respective activity.
Is this only in the home activity or also in other ones (say, issue list, issue details etc.)?

**Comment by Terrance** [comment]
Seems to happen with all of the top-level screens, including Search (whether a query has been entered or not; searches then don't go through whilst the 403 error is showing), Public Timeline and Trending Repos.

**Comment by Terrance** [comment]
I also seem to be getting frequent "API rate limit exceeded" errors whilst testing this -- may be related to #1013, if notifications can't be read with this token and they're being retried on all screen refreshes?

**Comment by Terrance** [comment]
More importantly, opening a link to a repo directly in the app also shows this 403 when the app first opens, and you can't pull-refresh or navigate around to make it load.

**Comment by maniac103** [comment]
Can you provide a stack trace from logcat including the 'Caused by...' parts?

**Comment by Terrance** [comment]
[03_01-08-17-23_224.log](https://github.com/slapperwan/gh4a/files/13816912/03_01-08-17-23_224.log)
[03_01-08-25-01_043.log](https://github.com/slapperwan/gh4a/files/13816925/03_01-08-25-01_043.log)

The first one is from opening the app via launcher, the second from opening a link to this repo.  Both of them do seem to suggest they're trying to refresh notifications (despite Notifications being disabled in Settings, though presumably that only affects the resulting Android notifications).

**Comment by EmanuelLoos** [comment]
Are there any news on this? I get "secondary rate limit exceeded" a lot lately.

**Comment by dtoxic47** [comment]
Also have  the problem "Rate limit..."
